deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,39 @@
+# Deploy SparkDotnet REPL to HDI Spark Cluster with Notebook Experience
+
+## Goal
+
+This documentation will elaborate the steps on how to deploy SparkDotnet REPL to HDI Spark Cluster and submit jobs through Jupyter Notebook using SparkDotnet.
+
+## Background
+
+We are currently using [dotnet-try](https://github.com/dotnet/try) as our dotnet REPL. ``` dotnet-try ``` can be used as a jupyter kernel and we implement it through [sparkmagic](https://github.com/jupyter-incubator/sparkmagic) and livy. To enable Jupyter Notebook Experience using SparkDotnet, this will include simple manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) in HDI Spark Cluster.
+
+## Step
+
+### 1. Go to Ambari, manually stop Livy Server on head node.
+
+Details to be added.
```

Comment:
I'll add some comments with the explanation I had written after working with you. Hopefully they help describe some of the details!
***
After creating an Azure HDInsight cluster (by visiting the [Azure Portal](portal.azure.com), + Create a Resource, and creating a new Azure HDInsight cluster resource), go to the Overview tab on your cluster. Ensure you've created a cluster with Spark 2.4 and HDI 4.0.

Go to Ambari by selecting "Ambari home" from your cluster's Overview page. 

Select "Spark2" from the left menu of Services. Next select "LIVY FOR SPARK2 SERVER," select the first head node (i.e. "hn0..."), and locate the "Livy for Spark2 Server" component. Select the three dots under "Action" and click Stop to stop Livy on the head node.